lib/verdict/experiment.rb

```diff
@@ -241,8 +243,9 @@ def disqualify_empty_identifier?
     @disqualify_empty_identifier
   end
 
-  def subject_qualifies?(subject, context = nil)
+  def subject_qualifies?(subject, dynamic_qualifiers, context = nil)
```

Comment:
We have to get better at documenting breaking changes like this 😅

Reply:
I’m on my mobile so can’t verify but if this is a public method, then we didn’t intend to make a breaking change here. cc @JonathanGin52

Reply:
The intention was for this change not to be breaking. I will address this issue, sorry for the inconvenience! If you don't mind me asking, what is the reason for calling this method directly?

Reply:
It's ok, the fix is very simple but it just caught me unaware from reading the Changelog displayed by Dependabot.

> If you don't mind me asking, what is the reason for calling this method directly?

We use the CookieStorage backend and recheck qualifications to ensure the cookie is still set and is valid.